### DISCUSSION TITLE
Having trouble with 1 particular collection not generating properly

### DISCUSSION THREAD

**Discussion by calebsw83** [meisnate12/Plex-Meta-Manager#862] [Q&A] [answered]
I had PMM up and working great, and then I saw a config on the github here that I wanted to add, but they used a different `sort_title` labeling scheme than I was, so I edited my templates to allow this new config to be placed where I wanted it. Previously the sort_title labels were using !A !B, etc, but now I was doing +A, +B, etc. The migration went smoothly except for one collection, and I cannot figure out why it won't work right. 

I have a collection of depressing/bleak movies called "The World is Hell" and to balance it out, I have a collection of uplifting movies called "The World is Not Hell." This "The World is not Hell" collection is the one I'm having troubles with. Here's the config info I had: 

```
templates:
  Special Genre:
    sort_title: "+A <<collection_name>>"
    smart_label: random
    collection_mode: show_items
collections:
  The World is Not Hell:
    template: {name: Special Genre}
    letterboxd_list: https://letterboxd.com/.../
```

After the migration, 2 things were happening that shouldn't have been. The collection's sort title was changed to simply "World is Not Hell" and the collection was not a smart collection. I checked the logs, and see this: 

```
Traceback (most recent call last):                                                                 
   File "/app/pmm/plex_meta_manager.py", line 512, in run_collection
     builder.load_collection()
   File "/app/pmm/modules/builder.py", line 2405, in load_collection
     self.obj = self.library.get_playlist(self.name) if self.playlist else self.library.get_collection(self.name)
   File "/app/pmm/modules/plex.py", line 725, in get_collection
     raise Failed(f"Plex Error: Collection {data} not found")
 modules.util.Failed: Plex Error: Collection The World is Not Hell not found
```

I have no idea what this means, but maybe someone here does. I decided to try to set this collection up manually because I realized that even if it worked as intended, it would still have a sort title starting with T instead of W, so I updated the collection info to this: 

```  
The World is Not Hell:
    sort_title: "+A World is Not Hell"
    collection_order: random
    #smart_label: random
    collection_mode: show_items
    letterboxd_list: https://letterboxd.com/.../
```

But no difference. I also changed to use the smart label as I originally intended. In all cases, the collections turns out to be a normal collection with the sort_title as "World is Not Hell." I'm not sure what I'm doing wrong. I tried deleting the collection and removing any metadata like labels and tags from all movies that have that assigned to them, but it didn't seem to help either. 

Any help would be greatly appreciated. Thanks. 

Edit: After re-reading my post, I wonder if it's possible to pull a config from the github page, but to give it a custom name? That might help, maybe it's the +A that's causing issues? IDK

**Comment by meisnate12** [comment]
could you post your meta.log so i can see whats going on

**Comment by calebsw83** [comment]
Yep, here you go! I've been running an abridged copy of my config so that I can try to work this issue out, and this log is from that and only shows details related to that specific collection [meta.log](https://github.com/meisnate12/Plex-Meta-Manager/files/8663394/meta.log)

**Comment by meisnate12** [comment]
have you deleted the collection and just let pmm make it?

**Comment by calebsw83** [comment]
I've been trying, but I feel like some remnants of it exist that I'm not seeing. I delete the collection, then I try filtering for any tags or labels that would have "World is Not Hell" and remove them as well. Yesterday, for the first time, I tried optimizing my plex database, hoping that it would remove any traces that were residing, but when I checked this morning, it still didn't run right. It's so bizarre, the config is identical to several other collections I have set up. It looks like it should probably run right, so I think the issue is with some remnants residing in plex, but I'm not sure how to approach this beyond what I've already done.

**Comment by calebsw83** [comment, marked as answer]
Whelp, now it's working again. I didn't even change anything from this morning, but I ran the script again in its entirety because I wanted to add some other categories, and now it's working right. Thanks for your help on this, wish I could tell you what changed in case other ppl have the same issue.
